ocamlformat.opam

```diff
@@ -28,6 +25,8 @@ depends: [
   "stdio" {< "v0.15"}
   "uuseg" {>= "10.0.0"}
   "uutf" {>= "1.0.1"}
+  "alcotest" {with-test}
+  "dune" {>= "2.8" & with-test & < "3.0"}
```

Comment:
That's why we had two `dune` in the depends for this package. The first one would be updated by Dune while generating the opam file and the other would be kept.

Reply:
Thanks, I added it back 🙂